contribs/gmf/src/authentication/Service.js

```diff
@@ -16,6 +16,7 @@ import olEventsEventTarget from 'ol/events/Target.js';
 
 /**
  * @typedef {Object} User
+ * @property {string|null} email User's email address
```

Comment:
```suggestion
 * @property {string} [email] User's email address
```
same for other properties...
Not completely sure but looks better :-)

Reply:
Same comment.